test/test_functional_tensor.py

```diff
@@ -868,6 +798,84 @@ def test_perspective_interpolation_warning(tester):
         tester.assertTrue(res1.equal(res2))
 
 
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+@pytest.mark.parametrize('size', [32, 26, [32, ], [32, 32], (32, 32), [26, 35]])
+@pytest.mark.parametrize('max_size', [None, 34, 40, 1000])
+@pytest.mark.parametrize('interpolation', [BILINEAR, BICUBIC, NEAREST])
+def test_resize(device, dt, size, max_size, interpolation, tester):
+
+    torch.manual_seed(12)
```

Comment:
do we have to set the seed? If yes maybe add a comment with a potential FIXME?

Reply:
I think we should create deterministic and random input instead purely random one

Reply:
OK, we generally don't do that, but I agree with you that it's better practice when we know the test aren't flaky.

Another option is to actually parametrize the seed, which we can do if the tests are very fast.

Reply:
Otherwise I would put it module-wise. Seed as parameter is something new to me :)

Reply:
I think this is already setting the seed module-wise, at least for the tests that will be run after this one.

In the long term we'll use something local like `rng = np.random.RandomState(0)` (surely there's a pytorch equivalent) and generate stuff from the local `rng` variable, but for now this is good enough

Reply:
For all tests that do not perform a loop and thus only need one seed per test invocation, we could use an [autouse fixture](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request) that is defined in the root `conftest.py`

```python
import pytest

@pytest.fixture(scope="function", autouse=True)
def random_seed():
    torch.manual_seed(12)
```